addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -19,6 +19,9 @@ const UrlPickerUserValueWidget = InputUserValueWidget.extend({
     custom_events: _.extend({}, InputUserValueWidget.prototype.custom_events || {}, {
         website_url_chosen: '_onWebsiteURLChosen',
     }),
+    events: { // Explicitely not consider all InputUserValueWidget events
+        'blur input': '_onInputBlur',
+    },
```

Comment:
I don't like this commit. It was working before. Nothing changed, except a potential chrome update. Can you give a real explanation of why this is needed? Because it is more code for less feature: the widget cannot be used in preview anymore (ok probably not needed for an URLPicker, but still).

Reply:
If you trigger an input event on an autocompleted field it will open the autocomplete. It is happening on Chrome as well as Firefox.
On Firefox tho it is not happening when selecting an autocomplete option but only when clicking away from it. 
Chrome reopen the autocomplete in both case: clicking away from the autocomplete or selecting an option.
Either Chrome is completely wrong or completely right. Firefox Half wrong half right on this...

So I think this commit is usefull.

But I recall now that removing the events was in fact not usefull... I must have forgoten it and left it there thinking it had a purpose... so in the end this is all we need to do:

```
-        $(this.inputEl).trigger('input');
+       this._value = this.inputEl.value;
```